test/afc/afc-specs.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai';
+import AfcService from '../../lib/afc';
+import { getServerWithFixtures, fixtures } from '../fixtures';
+
+
+chai.should();
+
+describe('afc', function () {
+  let server;
+  let socket;
+  let service;
+
+  afterEach(function () {
+    service.close();
+    if (server) {
+      server.close();
+    }
+  });
+
+  it('should create directory', async function () {
+    ({server, socket} = await getServerWithFixtures(fixtures.AFC_SUCCESS_RESPONSE));
```

Comment:
server and socket could be initialed in beforeEach function

Reply:
We pass different fixtures in every test. This is not possible in the current setup